src/http/server.rs

```diff
@@ -57,9 +57,15 @@ fn create_allowed_headers(headers: &HeaderMap, allowed: &BTreeSet<String>) -> He
 
   new_headers
 }
-pub async fn start_server(file_path: &String) -> Result<()> {
-  let server_sdl = fs::read_to_string(file_path)?;
-  let config = Config::from_sdl(&server_sdl)?;
+pub async fn start_server(file_path: Vec<String>) -> Result<()> {
+  let mut configs = Vec::new();
+  for file_path in &file_path {
+    let server_sdl = fs::read_to_string(file_path).expect("Failed to read file");
```

Comment:
Reading each file sequentially is going to be slow if there are many such files.